### PR TITLE
Remove usage of the `ember-awesome-macros` addon

### PR DESCRIPTION
**Pull request by Poltergeistz**
- Removed ember-awesome-macros from the project.
- Moved Personeelsbeheer Components to Octane.
- Fixed hbs lint errors related to the components.
- Fixed eslint errors.

Open for review and advises.

Also noticed that the EmployeePeriodSliceTable has a scrollbar inside causing the component to shrink in the browser. The issue seems to affect only desktop and above. 

Screenshot 2022-07-19 at 16-26-57 Personeelsbeheer Loket voor lokale besturen

### CODE REVIEW

File: app/components/personeelsbeheer/employee-period-slice-table.hbs [hunk 1]
```diff
@@ -3,23 +3,23 @@
   <table class="au-c-table au-c-table--zebra">
     <thead>
       <tr>
-        <th rowspan="3" style="min-width: 10rem; max-width: 13rem;">Niveau</th>
+        <th rowspan="3" style={{html-safe (concat "min-width: 10rem; max-width: 13rem;")}}>Niveau</th>
```
[the remark this answers is not in the document]
Comment (Windvis) [reply app/components/personeelsbeheer/employee-period-slice-table.hbs:6]: If it's done to workaround the `no-inline-styles` rule I would just disable that specific rule for this file. I believe we do that already in a couple of places.

**Review by Windvis — COMMENTED** [review]
quick partial review, I'll take a look at it again later but I have to go for now.

Thanks for working on this 👍.

File: app/components/personeelsbeheer/employee-observation-table-cell.js [hunk 2]
```diff
@@ -1,49 +1,69 @@
...
+      observations,
+      unitMeasure,
+      educationalLevel,
+      workingTimeCategory,
+      legalStatus,
+      sex,
+      onChange,
+    } = this.args;
+
+    this.handleOnChange = onChange;
```
[the remark this answers is not in the document]
Comment (Poltergeistz) [reply app/components/personeelsbeheer/employee-observation-table-cell.js:22]: > quick partial review, I'll take a look at it again later but I have to go for now.
> 
> Thanks for working on this +1.

Thanks for reviewing, I'll address the changes tomorrow +1

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
It seems there is a merge conflict on the .lint-todo file after merging #140. Could you rebase this branch on development to resolve it?

**Pull request merged**